### PR TITLE
Added new iter functionality

### PR DESCRIPTION
**Pull request by alangshur**

### CODE REVIEW

File: transpose/src/util/generic_model.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import json
```
Comment (Jon-Becker) [inline transpose/src/util/generic_model.py:1]: why you remove this?

File: transpose/src/util/generic_model.py [hunk 2]
```diff
@@ -13,6 +12,13 @@ def __init__(self, name: str='TransposeDataModel', data: object={}) -> None:
     # the data object should be returned when the object is converted to a dict
     def __dict__(self) -> dict:
         return self.data
+
+    def to_dict(self) -> dict:
+        return self.data
+
+    def __iter__(self) -> dict:
+        for k, v in self.data.items():
+            yield k, v
```
Comment (Jon-Becker) [inline transpose/src/util/generic_model.py:21]: very cool

File: transpose/src/util/generic_model.py [hunk 3]
```diff
@@ -44,5 +50,6 @@ def __getitem__(self, key: Any) -> Any:
     def __len__(self) -> int:
         return len(self.data)
     
-    def __dict__(self) -> dict:
-        return [d.__dict__() for d in self.data]
\ No newline at end of file
+    def __iter__(self) -> List[dict]:
+        for d in self.data:
+            yield dict(d)
```
Comment (Jon-Becker) [inline transpose/src/util/generic_model.py:55]: no newline at end of file!!!1!

File: transpose/src/util/generic_model.py (hunk 1 above)
Comment (alangshur) [reply transpose/src/util/generic_model.py:1]: not being used

**Comment by Jon-Becker** [comment]
image
breaking?!

**Pull request merged**